en/includes/guides/organization-management/try-a-b2b-use-case.md

```diff
@@ -174,79 +172,79 @@ On the {{ product_name }} Console, go to **Applications** and select the applica
 
 **API Authorization**
 
-1. Go to the **API Authorization** tab and authorize the APIs as follows.
+Go to the **API Authorization** tab and authorize the APIs as follows.
 
-    - For **Guardio Insurance Business App**
-      
-        - No API authorization is required.    
+- For **Guardio Insurance Business App**
+
+    - No API authorization is required.
+
+- For **Guardio Insurance Administrative App**
+
+    - Select the following APIs under **SYSTEM_ORG** category (organization level APIs).
```

Comment:
`SYSTEM_ORG` name is changed in UI

It should be Organization API. This need to be changed in IS and Asgardeo docs

Reply:
Will fix this with the PR for ]1]

[1] https://github.com/wso2/product-is/issues/19842